tgui/src/interfaces/faxmachine.ract

```diff
@@ -0,0 +1,30 @@
+<ui-display title='Authorization'>
```

Comment:
Recompile tgui!

Reply:
I am! I just need to sort out the other issues so i can at least get it to compile. It's on my todo list.

Reply:
Compiled